pages/common/git-magic.md

```diff
@@ -0,0 +1,29 @@
+# git magic
+
+> Automate add, commit, and push routines.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-magic>.
+
+- Commit changes with a generated message:
+
+`git magic`
+
+- Add untracked files and commit changes with a generated message:
```

Comment:
```suggestion
- [a]dd untracked files and commit changes with a generated message:
```

`[a]` (case-sensitive) is a command mnemonic. Learn more on our [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).

Reply:
I can't find anything about it in the style guide but I'll keep that in mind for the future. Thanks